### PR TITLE
ItemDB Update

### PR DESCRIPTION
**Pull request by Jedzkie**
- Added official effect for Thai Perfume CAST, Thai Perfume ASPD, Thai Perfume ATK and Thai Perfume MATK items.

### CODE REVIEW

File: src/map/status.c [hunk 1]
```diff
@@ -5953,8 +5967,10 @@ short status_calc_aspd_rate(struct block_list *bl, struct status_change *sc, int
 		aspd_rate += sc->data[SC_BATTLESCROLL]->val1 * 10;
 	if (sc->data[SC_STEAMPACK])
 		aspd_rate += sc->data[SC_STEAMPACK]->val2 * 10;
+	if (sc->data[SC_SKF_ASPD])
+		aspd_rate += sc->data[SC_SKF_ASPD]->val1 * 10;
```
Comment (dastgir) [inline src/map/status.c]: + means lower the aspd
- means increase the aspd
What's the behavior of this status?
I guess it should increase?

**Comment by hemagx** [comment]
the commit does include many syntax fixing, this is required indeed however it make it hard to be able to read the actual changes.
please separate syntax fixing in other PRs

**Comment by Jedzkie** [comment]
@hemagx okay, in my next pr.

File: src/map/status.c (hunk 1 above)
Comment (AnnieRuru) [inline src/map/status.c]: yes
that also means `SC_BATTLESCROLL` should be `-=` as well

**Pull request closed without merging**